runtime/oti/j9nonbuilder.h

```diff
@@ -5676,6 +5695,126 @@ typedef struct J9JavaVM {
 #define J9JAVAVM_CONTIGUOUS_HEADER_SIZE(vm) (J9JAVAVM_COMPRESS_OBJECT_REFERENCES(vm) ? sizeof(J9IndexableObjectContiguousCompressed) : sizeof(J9IndexableObjectContiguousFull))
 #define J9JAVAVM_DISCONTIGUOUS_HEADER_SIZE(vm) (J9JAVAVM_COMPRESS_OBJECT_REFERENCES(vm) ? sizeof(J9IndexableObjectDiscontiguousCompressed) : sizeof(J9IndexableObjectDiscontiguousFull))
 
+#if JAVA_SPEC_VERSION >= 16
+
+#define FFI_UPCALL_J9CLASS_EYECATCHER (UDATA)0x99669966
```

Comment:
This should be turned into a constant for general use.

Reply:
I removed this as it was temporarily used to check in an assertion which no longer exists.